pkg/apis/migration/migration.go

```diff
@@ -166,6 +171,7 @@ func GetVolumeMigrationService(ctx context.Context, volumeManager *cnsvolume.Man
 				stopCh := make(chan struct{})
 				informer.Informer().Run(stopCh)
 			}()
+			volumeMigrationInstance.cleanupStaleCRDInstances(ctx)
```

Comment:
If i understand correctly, there's one instance of `volumeMigrationService` in both controller and syncer. In that case, we're going to have two cleanup goroutines running? Does it make sense to add this functionality to full sync, since its the same model?

Reply:
controller and syncer run in two diff containers, so we have one instance per container.
it will be good if we run this clean up routine in only one container as @RaunakShah has suggested.

Reply:
Can you supply control flag in GetVolumeMigrationService to decide if volumeMigrationInstance.cleanupStaleCRDInstances() routin needs to be called or not.

when called from controller init supply this flag as `false` and when called from syncer init supply this flag is `true`.

Reply:
+1

Reply:
```
SYNCER

# kubectl logs vsphere-csi-controller-587b968b4b-vbtzj -c vsphere-syncer -n kube-system | grep "Triggering"
2020-10-20T21:55:19.830Z	INFO	migration/migration.go:438	Triggering CnsVSphereVolumeMigration cleanup routine	{"TraceId": "683051b7-f7ed-46bb-bbc9-13aeb5704652"}
2020-10-20T21:56:19.830Z	INFO	migration/migration.go:438	Triggering CnsVSphereVolumeMigration cleanup routine	{"TraceId": "5316ce1e-2455-48df-9da1-d4daaebc078a"}
2020-10-20T21:57:19.829Z	INFO	migration/migration.go:438	Triggering CnsVSphereVolumeMigration cleanup routine	{"TraceId": "a52c0079-d650-49d6-a87e-ec3a2a937848"}
...
...
2020-10-20T22:52:19.830Z	INFO	migration/migration.go:438	Triggering CnsVSphereVolumeMigration cleanup routine	{"TraceId": "b03c3f1d-51ee-46e7-bd6b-976eccea7c96"}


CONTROLLER
root@k8s-master:~# kubectl logs vsphere-csi-controller-587b968b4b-vbtzj -c vsphere-csi-controller -n kube-system | grep "Triggering"
No results found
root@k8s-master:~#